_posts/en/newsletters/2023-06-14-newsletter.md

```diff
@@ -0,0 +1,190 @@
+    delays its confirmation.  Because Bitcoin Core and other full nodes
+    don't currently relay transactions that contain annexes, this is not
+    a problem at present.  Jager [replied][jager annex4] that he wants
+    to store signatures from ephemeral keys for a type of [vault][topic
+    vaults] that doesn't require a soft fork, and he  [suggested][jager
+    annex3] that some
+    [previous work][bitcoin core #24007] in Bitcoin Core could possibly
+    address the problem with annex relay in some multiparty protocols.
+
+- **Draft BIP for silent payments:** Josie Baker and Ruben Somsen
+  [posted][bs sp] to the Bitcoin-Dev mailing list a draft BIP for
+  [silent payments][topic silent payments], a type of reusable payment
+  code that will produce a unique onchain address each time it is used,
+  preventing [output linking][topic output linking] that can
+  significantly reduce the privacy of users (including users not
+  directly involved in a transaction).  The draft goes into detail about
+  the benefits of the proposal, its tradeoffs, and how software can
+  effectively use it.  Several insightful comments have already been
+  posted on the [PR][bips #1458] for the BIP.
+
+## Waiting for confirmation #5: Policy for Protection of Node Resources
+
+_A limited weekly [series][policy series] about transaction relay,
+mempool inclusion, and mining transaction selection---including why
+Bitcoin Core has a more restrictive policy than allowed by consensus and
+how wallets can use that policy most effectively._
+
+{% include specials/policy/en/05-dos.md %}
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Allow inbound whitebind connections to more aggressively evict peers when slots are full][review club 27600]
+is a PR by Matthew Zipkin (pinheadmz) that improves a node operator's
+ability in certain cases to configure desired peers for the node.
+Specifically, if the node operator has whitelisted a potential inbound
+peer (for example, a light client controlled by the node operator), then
+without this PR, and depending on the node's peer state, it's possible
+that the node will deny this light client's connection attempt.
+
+This PR makes it much more likely that the desired peer will be able to
+connect to our node. It does this by evicting an existing inbound peer
+that, without this PR, would have been ineligible for eviction.
+
+{% include functions/details-list.md
+  q0="Why does this PR only apply to inbound peer requests?"
+  a0="Our node _initiates_ outbound connections; this PR modifies how
+      the node _reacts_ to an incoming connection request.
+      Outbound nodes can be evicted, but that's done with an entirely
+      separate algorithm."
+  q0link="https://bitcoincore.reviews/27600#l-33"
```

Comment:
```suggestion
  a0link="https://bitcoincore.reviews/27600#l-33"
```

Reply:
Fixing and also adding test to automatically catch this wrong-parameter name typos in the future.